src/widgets/settingspages/IgnoresPage.cpp

```diff
@@ -49,7 +49,7 @@ void addPhrasesTab(LayoutCreator<QVBoxLayout> layout)
                     ->initialized(&getSettings()->ignoredMessages))
             .getElement();
     view->setTitles(
-        {"Pattern", "Regex", "Case Sensitive", "Block", "Replacement"});
+        {"Pattern", "User", "Regex", "Case Sensitive", "Block", "Replacement"});
```

Comment:
Line 44 (that github doesn't let me mark)
You should add a comment says something like: "Leaving the User tab empty will make it target all users"